recipes/legit/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set packagename = 'legit' %}
+{% set version = "0.2.0" %}
+{% set md5 = "5df3cd402b0de8f0251a87904b5eb88c" %}
+
+package:
+  name: {{ packagename }}
+  version: {{ version }}
+
+source:
+  fn: {{ packagename }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ packagename[0] }}/{{ packagename }}/{{ packagename }}-{{ version }}.zip
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - legit = legit.cli:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - clint >=0.4.1
+    - gitpython >=0.3.4
+    - six >=1.9.0
+  run:
+    - python
+    - clint >=0.4.1
+    - gitpython >=0.3.4
+    - six >=1.9.0
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  imports:
+    - {{ packagename }}
+
+  commands:
+    - legit --help
```

Comment:
The `legit` cli needs a git repository to work off of or you just get this message:

```
$ legit
Not a git repository.
```
and the real kicker is that it [exits 128] (https://github.com/kennethreitz/legit/blob/v0.2.0/legit/scm.py#L47), which means that your test section will always fail if you dont have a git repository.

Another option would be to change the source section to be:

```yaml
source:
  git_url: https://github.com/kennethreitz/legit
  git_tag: v{{ version }}
```

and then change the commands section to be:

```yaml
commands:
  - cd {{ environ.SRC_DIR }} && legit --help
```

But *that* [exits 1] (https://github.com/kennethreitz/legit/blob/v0.2.0/legit/cli.py#L53) after printing the help message 

So do the [`-v` or `--version` flags] (https://github.com/kennethreitz/legit/blob/v0.2.0/legit/cli.py#L57).

I guess the best option is to just to use `legit help`, which exercises the CLI and prints out:

```
>legit help
help <command>

Display help for legit command.
```

and, most importantly, exits with a 0, so it does not cause conda build to fail!


-------

@jakirkham this seems like a reasonable exception to the "don't clone the git repo" rule, right?

tl; dr

source should look like:
```yaml
source:
  git_url: https://github.com/kennethreitz/legit
  git_tag: v{{ version }}
```

commands should look like:

```yaml
commands:
  - cd {{ environ.SRC_DIR }} && legit --help
```